www/json/connectionConfig.json

```diff
@@ -0,0 +1,15 @@
+{
+    "connectUrl": "http://52.91.65.149:8080",
+    "android": {
+        "auth": {
+            "method": "token_list",
```

Comment:
there is no `token_list` method supported on the phone. See https://github.com/e-mission/e-mission-phone/pull/465 for examples